Build break in libcurl dependency

On Xcode 11 Beta 5:

```
FAILED: ../deps/build/libcurl/src/ex_libcurl-stamp/ex_libcurl-build ../deps/local/lib/libcurl.a 
cd /Users/zach/turicreate/deps/build/libcurl/src/ex_libcurl-build && make && /usr/local/Cellar/cmake/3.15.2/bin/cmake -E touch /Users/zach/turicreate/deps/build/libcurl/src/ex_libcurl-stamp/ex_libcurl-build
Making all in lib
/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-am
Making all in src
Making all in ../docs
Making all in .
make[3]: Nothing to be done for `all-am'.
Making all in cmdline-opts
make[3]: Nothing to be done for `all'.
  CC       curl-tool_hugehelp.o
tool_hugehelp.c:27:6: error: redefinition of 'hugehelp'
void hugehelp(void)
     ^
tool_hugehelp.c:20:6: note: previous definition is here
void hugehelp(void) {}
     ^
1 error generated.
make[2]: *** [curl-tool_hugehelp.o] Error 1
```

--- a/deps/cmake/ExternalProjectLibCurl.cmake
+++ b/deps/cmake/ExternalProjectLibCurl.cmake
@@ -18,7 +18,7 @@ ExternalProject_Add(ex_libcurl
   PREFIX ${CMAKE_SOURCE_DIR}/deps/build/libcurl
   URL ${CMAKE_SOURCE_DIR}/deps/src/curl-7.65.1
   INSTALL_DIR ${CMAKE_SOURCE_DIR}/deps/local
-  CONFIGURE_COMMAND env CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER} "CFLAGS=-fPIC ${ARCH_FLAG} ${C_REAL_COMPILER_FLAGS}" <SOURCE_DIR>/configure --prefix=<INSTALL_DIR> --without-winidn --without-libidn --without-libidn2 --without-nghttp2 --without-ca-bundle --without-polarssl --without-cyassl --without-nss --disable-crypto-auth --enable-shared=no --enable-static=yes --disable-ldap --without-librtmp --without-zlib --libdir=<INSTALL_DIR>/lib ${EXTRA_CONFIGURE_FLAGS}
+  CONFIGURE_COMMAND env CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER} "CFLAGS=-fPIC -w ${ARCH_FLAG} ${C_REAL_COMPILER_FLAGS}" "CXXFLAGS=-w" <SOURCE_DIR>/configure --prefix=<INSTALL_DIR> --without-winidn --without-libidn --without-libidn2 --without-nghttp2 --without-ca-bundle --without-polarssl --without-cyassl --without-nss --disable-crypto-auth --enable-shared=no --enable-static=yes --disable-ldap --without-librtmp --without-zlib --libdir=<INSTALL_DIR>/lib ${EXTRA_CONFIGURE_FLAGS}
   BUILD_BYPRODUCTS ${CMAKE_SOURCE_DIR}/deps/local/lib/libcurl.a
   )
 

